deploy/deploy-hyperlane.md

```diff
@@ -178,35 +177,176 @@ $  head -n 22 artifacts/agent_config.json
   },
 ```
 
-## 3. Run validators
+## 3. Run validators and relayers
 
-Follow the [validators](../operators/validators/ "mention") guide to run validators for the [messaging.md](../protocol/messaging.md "mention")on your local chain. These validators will provide the security for messages sent **from** your chain **to** remote chains.
+We've partnered with [Kurtosis](https://www.kurtosis.com/) to offer a smooth validator and relayer deployment experience through [Kurtosis Cloud](https://cloud.kurtosis.com/). This section of the guide will walk you through how to deploy a validator and relayer using the no-code, Kurtosis Cloud UI that connects a Goerli to Polaris. 
 
-Include the agent config from the [#2.-deploy-contracts](deploy-hyperlane.md#2.-deploy-contracts "mention") step in `CONFIG_FILES`. If you're using Docker, you will need to mount the file into the container.
+If you wish to read more about validators and relayers, check out our docs on these topics at [Validators](../operators/validators/ "mention") and [Relayers](../operators/relayers/ "mention").
 
-You should have already set up your validator keys in step [#1.-setup-keys](deploy-hyperlane.md#1.-setup-keys "mention"), so you can skip that part of the [validators](../operators/validators/ "mention") guide.
+For feedback or questions about using Kurtosis in this section, please file a [Github Issue](https://github.com/kurtosis-tech/kurtosis/issues/new/choose) or start a [Github Discussion](https://github.com/kurtosis-tech/kurtosis/discussions/new?category=q-a). Alternatively, the Kurtosis team is available for technical support on their [Discord server](https://discord.com/invite/jJFG7XBqcY) as well.
 
-{% hint style="warning" %}
-Make sure these validators match the addresses you provided when in your `MultisigIsmConfig`. Otherwise, the [multisig-ism.md](../protocol/sovereign-consensus/multisig-ism.md "mention")s you deployed in the previous step will not be able to verify messages sent from your chain.
-{% endhint %}
+### Log in to Kurtosis Cloud to provision an instance
 
-{% content-ref url="../operators/validators/" %}
-[validators](../operators/validators/)
-{% endcontent-ref %}
+To begin, visit the [Kurtosis Cloud website](https://cloud.kurtosis.com/) (direct link: https://cloud.kurtosis.com/) and sign in with Google. 
 
-## 4. Run relayer
+If this is the first time you're using Kurtosis Cloud, Kurtosis will automatically begin provisioning a remote cloud instance for your validator and relayer (see screenshot below). This should take no longer than 3 minutes.
```

Comment:
`This should take no longer than 3 minutes.`
Nit, I'd be less specific with the time: `Provisioning should take only a few minutes`

Reply:
```suggestion
If this is the first time you're using Kurtosis Cloud, Kurtosis will automatically begin provisioning a remote cloud instance for your validator and relayer (see screenshot below). Provisioning should take only a few minutes.
```